docs/usage/private-modules.md

```diff
@@ -212,6 +212,8 @@ For instructions on this, see the above section on encrypting secrets for the Wh
 - Use the resulting HTML encrypt page to encrypt secrets for your app before adding them to user/repository config
 - Configure the app to run with `privateKey` set to the private key you generated above
 
+Note: Encrypted values can't be used in the "AdminBot config".
```

Comment:
"admin config"

Reply:
I used the spelling from above, however now updated in both places from `AdminBot config` to `Admin/Bot config`